editor.planx.uk/src/pages/FlowEditor/lib/store/shared.ts

```diff
@@ -13,6 +13,7 @@ export interface SharedStore extends Store.Store {
   flow: Store.flow;
   flowSlug: string;
   flowName: string;
+  flowMetabaseLink: string;
```

Comment:
As this is a nullable value (for flows where it's not set) it might be best to type it as `string | null` and set the initial value to `null`.

<img width="995" alt="image" src="https://github.com/theopensystemslab/planx-new/assets/20502206/388122e1-bbb1-4ef9-87ee-7031a3f00eb1">

Reply:
Fixed here: 6ecabd07f9a64a721b55e274c44c714f8820c53c